plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -140,7 +139,7 @@ func ClusterRoles() []rbac.ClusterRole {
 
 				// additional admin powers
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("localsubjectaccessreviews").RuleOrDie(),
-				rbac.NewRule(ReadWrite...).Groups(rbacGroup).Resources("roles", "rolebindings").RuleOrDie(),
+				rbac.NewRule(ReadWrite...).Groups(rbac.GroupName).Resources("roles", "rolebindings").RuleOrDie(),
```

Comment:
I don't want the rules to use the groupNames from the API packages.  For other packages, no dependency currently exists and we don't want one.  I'd keep this package consistent.